depends/packages/libmultiprocess.mk

```diff
@@ -9,6 +9,7 @@ $(package)_dependencies += native_capnp
 endif
 
 define $(package)_set_vars :=
+$(package)_config_opts += -DCMAKE_INSTALL_LIBDIR=lib/
```

Comment:
> Rather than adding lib64/ to the pkg-config and link flags, I opted for always installing into lib/.

Mind sharing your reasoning behind this decision? Asking because the alternative looks more generic and will work if any other dependency package will switch to CMake.

---

A side note, not directly related to this PR: the CMake-based build system has no such an issue at all.

Reply:
> > Rather than adding lib64/ to the pkg-config and link flags, I opted for always installing into lib/.
> 
> Mind sharing your reasoning behind this decision?

The quote you're responding is from fanquake, but I think it makes sense for the depends build to hardcode "lib" here because it is already hardcoding "lib" in `config.site.in`:

https://github.com/bitcoin/bitcoin/blob/d752349029ec7a76f1fd440db2ec2e458d0f3c99/depends/config.site.in#L91

So this change just keeps the build consistent and ensure the bitcoin configure script can find the library built by depends on all platforms.

> Asking because the alternative looks more generic and will work if any other dependency package will switch to CMake.

This change isn't very clean so I agree there are probably better alternatives, but I'm actually not sure which alternative you're referring to here. I do agree it should become simpler when more of the build switches to cmake, though.